packages/git-fetcher/src/index.ts

```diff
@@ -3,6 +3,7 @@ import { Cafs, DeferredManifestPromise } from '@pnpm/fetcher-base'
 import preparePackage from '@pnpm/prepare-package'
 import rimraf from '@zkochan/rimraf'
 import execa from 'execa'
+import { URL } from 'url'
 
 export default () => {
```

Comment:
probably it should be passed through options here

and converted to a set.

Reply:
thanks for the refactor!